site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -66,6 +66,52 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
 [gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
 
+### Choosing a GKE cluster mode
+
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+Agones supports GKE Standard mode and has alpha support for GKE Autopilot mode.
+You can't migrate Agones game servers between Standard and Autopilot clusters.
```

Comment:
I don't think that reads quite right. We designed the Autopilot functionality so that e.g. you could take a `Fleet` and run it on an Autopilot cluster without much fuss (except the considerations below). What you can't do is migrate a GKE Standard cluster to Autopilot.

For companies with e.g. blue/green deployment and decent DevOps, this could literally be as simple as spinning up a new set of clusters using a different terraform chart, since we have multicluster allocation.

Maybe something like: 

`Fleet` and `GameServer` specifications that worked previously on Standard clusters are compatible with Autopilot, subject to the [considerations below](link to consideration)

Reply:
Oh wow haha the internal UG made me immediately think that migrating the Agones workloads was impossible. Thanks for clearing it up